packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -145,15 +146,17 @@ export const Headline = styled.h3`
     if (!script) {
       return '';
     }
-
     return topStory ? getParagon(script) : getPica(script);
   }}
   color: ${C_EBON};
   ${({ service }) => getSerifMedium(service)}
   margin: 0; /* Reset */
   padding-bottom: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    ${props => (props.script ? getParagon(props.script) : '')};
+    ${props =>
```

Comment:
I think when dealing with props it's preferable to destructure e.g. `({ script, topStory }) =>`

Reply:
I had already created an issue on this here is the link. https://github.com/bbc/psammead/issues/1921

Reply:
@jroebu14 Could you also leave your thoughts on https://github.com/bbc/psammead/issues/1921, please?